File: hw/chiriboga/u1/hw09CalculateIncomeTax/CalculateIncomeTax/src/ec/edu/espe/calculateincometax/menu/Menu.java
package ec.edu.espe.calculateincometax.menu;

import java.util.Scanner;
import static utils.Manage.createEmployee;
import static utils.Manage.readDataEmployee;
import static utils.Manage.deleteEmployee;

/**
 *
 * @author Kerlly Chiriboga, ODS
 */
public class Menu {
    public static void showMenu() {
        try (Scanner scanner = new Scanner(System.in)) {
            int operation;

            do {
                System.out.println("Menu:");
                System.out.println("1. Create Employee");
                System.out.println("2. Read Employee");
                System.out.println("3. Delete Employee");
                System.out.println("4. Exit");

                System.out.print("Enter the option: ");
                operation = scanner.nextInt();
                scanner.nextLine();

                switch (operation) {
                    case 1:
                        createEmployee(scanner);
                        break;
                    case 2:
                        readDataEmployee();
                        break;
                    case 3:
                        deleteEmployee(scanner);
                        break;
                    case 4:
                        System.out.println("Exiting...");
                        break;

                    default:
                        System.out.println("Invalid operation.");
                }
            } while (operation != 4);
        }
    }
}

File: hw/chiriboga/u1/HomeworksUnit1/hw09CalculateIncomeTax/CalculateIncomeTaxPython/ec/edu/espe/calculateincometax/menu/Menu.py
from utils.Manage import create_employee
from utils.Manage import read_data_employee
from utils.Manage import delete_employee

class showMenu:
    while True:
        print("Menu")
        print("1. Create Employee")
        print("2. Read Employees")
        print("3. Delete Employee")
        print("4. Exit")

        try: 
            operation = (int(input("Enter the operation: ")))

            if operation == 1:
                create_employee()     
            elif operation == 2:
                read_data_employee()
            elif operation == 3:
                delete_employee()
            elif operation == 4:
                print("Exiting...")
                break
            else: 
                print("Invalid Operation. Please enter a number between 1 and 4. ")

        except ValueError:
            print("Invalid input. Please enter a valid number")

showMenu()
            
